soccer/Processor.cpp

```diff
@@ -483,6 +449,7 @@ void Processor::run() {
 
         // Send motion commands to the robots
         sendRadioData();
+        _manual_control_node->applyControlsToRobots(&_context.state.self);
```

Comment:
Should this be in the manual control node `run` function?

Reply:
yeah, I think so